rllib/evaluation/tests/test_agent_collector.py

```diff
@@ -332,6 +332,19 @@ def test_view_requirement_with_shfit_step(self):
         # check if the padding in the beginning is correct
         check(sample_batch["prev_obses"][0], np.ones((3, 1)) * expected_obses[0])
 
+    def test_begin_with_inference(self):
+        # Test if AC is able to build for inference immediately
+        obs_space = gym.spaces.Box(-np.ones(4), np.ones(4))
+        view_reqs = {
+            SampleBatch.T: ViewRequirement(SampleBatch.T),
+            SampleBatch.REWARDS: ViewRequirement("reward", space=obs_space),
```

Comment:
space of reward should just be 1 dimensional? why is it called obs_space?

Reply:
My mistake, changed. Thanks